compio-log/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+#[cfg_attr(not(feature = "enable_log"), doc(hidden))]
+pub use tracing::*;
+
+#[cfg(not(feature = "enable_log"))]
+pub mod dummy;
+
+#[cfg(feature = "enable_log")]
+#[macro_export]
+macro_rules! instrument {
```

Comment:
This macro should not be used in async fn, right?

Reply:
yes